Car.h

```diff
@@ -1,9 +1,16 @@
 #pragma once
 
-struct Car : public Vehicle
+#include "Vehicle.h"
+
+struct Car : Vehicle
 {
     Car(const std::string& s);
+
+    virtual ~Car() override;
+    Car(const Car&);
+    Car& operator=(const Car&) = default;
+
```

Comment:
move this copy-assign operator implementation to the cpp file, with the other implementations.
- [ ] revise